src/main/java/frc/robot/PowercellDetection.java

```diff
@@ -0,0 +1,77 @@
+package frc.robot;
+
+import java.util.Arrays;
+import java.util.Comparator;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class PowercellDetection {
+
+    private final double SCREEN_AREA = 1; //TODO: Determine area camera view.
+
+    private int nb;
+    private double[] boxes;
+    private Target[] targets;
+
+    private Drivetrain drive;
```

Comment:
I think it would be better to not interact with the drivetrain in this class. It should be enough to just hand a heading and distance to the robot, who can use the drivetrain to go to the target.

Reply:
Actually, just have a function to return either one or more `Target`s. Then the robot main class can do all kinds of things with it, like draw boxes on the camera feed (maybe) that aren't just driving towards the nearest target. This is a better fit with the name of the class, it just detects game pieces, it doesn't "act" on it.

Reply:
Added a getter for targets and moved code for moving the robot based on target position to drivetrain. I believe that should be better.